src/tools/rados/rados.cc

```diff
@@ -1428,7 +1435,10 @@ static int do_get_inconsistent_cmd(const std::vector<const char*> &nargs,
     }
     items.clear();
   }
-  formatter.flush(cout);
+  if (opened) {
+    formatter.close_section();
```

Comment:
we do close_section() at https://github.com/ceph/ceph/pull/7289/commits/ee97af5d12cd6c34d3c7aae824b6446450460f60#diff-c24acfa82f15c38ae9370cab8630a579R1430, so it's not necessary?